src/lib/Sympa/HTMLDecorator.pm

```diff
@@ -230,6 +232,30 @@ sub decorate_email_at {
     return $decorated;
 }
 
+sub decorate_email_gecos {
+    my $self = shift;
+
+    my $decorated = '';
+    my $email_re  = Sympa::Regexps::addrspec();
+    my $language  = Sympa::Language->instance;
+    while (my $item = $self->_queue_shift) {
+        if ($item->{event} eq 'text') {
+            my $dtext = Sympa::Tools::Text::decode_html($item->{text});
+            if ($dtext =~ s{<?\b($email_re)\b>?}{}g) {
```

Comment:
```
            if ($dtext =~ s{<?\b($email_re)\b>?}{}g) {
```
E-mail and similar texts will entirely disappear.  Code such as
```
            my $replacement = $language->gettext('address@concealed');
            if ($dtext =~ s{\b$email_re\b}{$replacement}g) {
```
may be better.

Reply:
:+1: 
I will update my PR this week

Reply:
Done.

Reply:
I confirmed.